uxarray/__init__.py

```diff
@@ -1,2 +1,14 @@
 from .grid import *
 from .helpers import *
+# Sets the version of uxarray currently installed
+# Attempt to import the needed modules
+try:
+    from importlib.metadata import version as version
+except Exception:
+    from importlib_metadata import version as version
```

Comment:
Okay, last change before I'll be okay with approving.

```suggestion
try:
    from importlib.metadata import version as _version
except Exception:
    from importlib_metadata import version as _version
```

It will be better to specify this import as an underscored method. Since this is no longer in a function, things imported in this addition will be available to import under the uxarray namespace.

Like it is now, uxarray will appear to have a `ux.version()` and a `ux.__version__`. If we change the import to `ux._version`, it will be more obvious that that's not the one intended for public use. 


I installed this on my machine to demonstrate and this is what I see:
``` ipython
>>> import uxarray as ux
>>> ux.version
<function version at 0x112c01750>
>>> ux.__version__
'2023.5.0'
```

Reply:
That is a very good point, I see what you mean.